doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1566,7 +1567,7 @@ It must follow the same representation conventions as context attribute metadata
 A `subject` contains the following subfields:
 
 + `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or type must be present):
-    + `id` or `idPattern`: Id or pattern of the affected entities (optional). Both cannot be used at the same time.
+    + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same time, but at least one of them must be set.
```

Comment:
Lookig for the string (e.g. grep) 

```
`id` or `idPattern`
```
there are 4 occurences in the .apib file. Thus, I understand that this PR should also include the 2 missing ones at this moment.